From a7ba6e685392534f33237f92f7893291c8e3049b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Wed, 28 Nov 2012 13:21:33 +0800
Subject: [PATCH] Revert "rk292x (v86 & tr726) :add sd-vcc control while sd hw
 initial"

This reverts commit 64a1ccaaecab8c4e6c1360877c66a6980a7e5332.

Only revert block/blk-core.c and fs/fat/misc.c change.
---
 block/blk-core.c | 11 +----------
 fs/fat/misc.c    |  2 --
 2 files changed, 1 insertion(+), 12 deletions(-)

diff --git a/block/blk-core.c b/block/blk-core.c
index 6f80fe23587e..b7cd1a4f3d84 100755
--- a/block/blk-core.c
+++ b/block/blk-core.c
@@ -2037,18 +2037,9 @@ bool blk_update_request(struct request *req, int error, unsigned int nr_bytes)
 			error_type = "I/O";
 			break;
 		}
-#if 0
-#if defined(CONFIG_SDMMC_RK29) && !defined(CONFIG_SDMMC_RK29_OLD)  //Modifyed by xbw at 2011-21-14
-        printk(KERN_DEBUG "end_request: %s error, dev %s, sector %llu\n",
-               error_type, req->rq_disk ? req->rq_disk->disk_name : "?",
-               (unsigned long long)blk_rq_pos(req));
-
-#else
-		printk(KERN_ERR "end_request: %s error, dev %s, sector %llu\n",
+		printk(KERN_DEBUG "end_request: %s error, dev %s, sector %llu\n",
 		       error_type, req->rq_disk ? req->rq_disk->disk_name : "?",
 		       (unsigned long long)blk_rq_pos(req));
-#endif
-#endif      
 	}
 
 	blk_account_io_completion(req, nr_bytes);
diff --git a/fs/fat/misc.c b/fs/fat/misc.c
index c168c27bb709..6d93360ca0cc 100644
--- a/fs/fat/misc.c
+++ b/fs/fat/misc.c
@@ -50,7 +50,6 @@ EXPORT_SYMBOL_GPL(__fat_fs_error);
  */
 void fat_msg(struct super_block *sb, const char *level, const char *fmt, ...)
 {
-        #if 0
 	struct va_format vaf;
 	va_list args;
 
@@ -59,7 +58,6 @@ void fat_msg(struct super_block *sb, const char *level, const char *fmt, ...)
 	vaf.va = &args;
 	printk("%sFAT-fs (%s): %pV\n", level, sb->s_id, &vaf);
 	va_end(args);
-        #endif
 }
 
 /* Flushes the number of free clusters on FAT32 */
-- 
2.35.3

